pkg/l4lb/config.go

```diff
@@ -0,0 +1,55 @@
+package l4lb
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/containernetworking/cni/pkg/types"
+
+	"github.com/dcos/dcos-cni/pkg/minuteman"
+)
+
+type NetConf struct {
+	types.NetConf
+	Spartan   bool                   `json:"spartan, omitempty"`
```

Comment:
could this be more clearly named `SpartanEnabled`? (if that's what it really means...)